docs/guide/03-styling-templates.md

```diff
@@ -15,6 +15,48 @@ To help with dynamic styling, lit-html provides two directives for manipulating
 *   [`classMap`](template-reference#classmap) sets classes on an element based on the properties of an object.
 *   [`styleMap`](template-reference#stylemap) sets the styles on an element based on a map of style properties and values.
 
+## Setting classes with classMap {#classmap}
+
+Like `styleMap`, the `classMap` directive lets you set a group of classes based on an object.
+
+```js
+import {html} from 'lit-html';
+import {classMap} from 'lit-html/directives/class-map.js';
+
+const itemTemplate = (item) => {
+  const classes = {selected: item.selected};
+  return html`<div class="menu-item ${classMap(classes}">Classy text</div>`
```

Comment:
Missing `)` after `classes`, and `;` at end of line

Reply:
D'oh. Saw this after merging. Sending in a followup PR.